packages/wallet-connectors/package.json

```diff
@@ -20,7 +20,7 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@concordium/browser-wallet-api-helpers": "^2.0.0",
+        "@concordium/browser-wallet-api-helpers": "^2.4.0",
```

Comment:
It looks like this upgrade necessitates changes to `signAndSendTransaction`:
```
Argument of type 'Record<string, unknown>' is not assignable to parameter of type 'SmartContractParameters'.
```

Reply:
@DOBEN I've found that doing this part properly is a bigger change that I've done separately. Please see https://github.com/Concordium/concordium-dapp-libraries/pull/34.